libs/execution_context/ebpf_program.c

```diff
@@ -105,27 +108,49 @@ _ebpf_program_program_info_provider_changed(
 
         ebpf_program_data_t* program_data = (ebpf_program_data_t*)provider_data->data;
         if (program_data == NULL) {
+            EBPF_LOG_MESSAGE_GUID(
+                EBPF_TRACELOG_LEVEL_ERROR,
+                EBPF_TRACELOG_KEYWORD_PROGRAM,
+                "An extension cannot have empty program_data",
+                program->parameters.program_type);
```

Comment:
Should we also print some information that identifies that extension? (may be provider GUID)?

Reply:
program->parameters.program_type is the program info provider GUID.